libweasyl/libweasyl/models/tables.py

```diff
@@ -633,7 +633,6 @@ def default_fkey(*args, **kwargs):
     Column('sessionid', String(length=64), primary_key=True, nullable=False),
     Column('created_at', ArrowColumn(), nullable=False, server_default=text('now()')),
     Column('userid', Integer()),
-    Column('csrf_token', String(length=64)),
```

Comment:
We may be missing the alembic migration to remove the `csrf_token` column here.

```
weasyl=# select * from sessions;
       sessionid        |         created_at         | userid | csrf_token | additional_data |    ip_address     | user_agent_id 
------------------------+----------------------------+--------+------------+-----------------+-------------------+---------------
 SOr3kDbrUyiKp-L2KbpuqA | 2021-07-17 23:23:26.675837 |  26707 |            |                 | ::ffff:172.18.0.1 |             1
(1 row)

weasyl=# \q
could not save history to file "/root/.psql_history": No such file or directory
root@d77f21a0be0b:/# exit
kyra@kitty:~/git/weasyl$ git log
commit f0c0c62b412a1c4ba4f1d4b43fd62c546ab1f8a6 (HEAD, charmander/origin-csrf)
Author: Charmander <~@charmander.me>
Date:   Tue Jul 13 00:23:39 2021 -0700
```

Reply:
> Intentionally missing a migration to delete `csrf_token` for now, just in case we want to undo this.

I promise to put it in right after :D

Reply:
Missed that. My bad!